workshops/test/test_autoupdate_profile.py

```diff
@@ -1,21 +1,28 @@
 from django.core.urlresolvers import reverse
 
-from workshops.models import Person, Lesson, Qualification
+from workshops.models import Person, Lesson, Qualification, KnowledgeDomain
 from workshops.test import TestBase
 
 
 class TestAutoUpdateProfile(TestBase):
     def setUp(self):
         self._setUpAirports()
+
         self.user = Person.objects.create_user(
             username='user', personal='', family='',
             email='user@example.org', password='pass')
+
         self.python = Lesson.objects.create(name='swc/python')
         self.shell = Lesson.objects.create(name='swc/shell')
         self.matlab = Lesson.objects.create(name='swc/matlab')
         self.git = Lesson.objects.create(name='swc/r')
```

Comment:
There's `self._setUpLessons()` avaliable (defined in `base.py`), which you can perhaps extend to add two other lessons.